jupyterlab/labapp.py

```diff
@@ -536,8 +537,32 @@ class LabApp(NBClassicConfigShimMixin, LabServerApp):
         help="Whether to expose the global app instance to browser via window.jupyterlab",
     )
 
-    collaborative = Bool(
-        False, config=True, help="Whether to enable collaborative mode (experimental)."
+    collaborative = Bool(False, config=True, help="Whether to enable collaborative mode.")
+
+    collaborative_file_poll_interval = Int(
+        1,
+        config=True,
+        help="""The period in seconds to check for file changes in the back-end (relevant only
+        in collaborative mode). Defaults to 1s, if 0 then file changes will only be checked when
+        saving changes from the front-end.""",
+    )
+
+    collaborative_document_cleanup_delay = Int(
+        60,
```

Comment:
How much overhead is there in reloading the document into memory? 60s seems on the short side as users could easily talk for 1m without making changes. Maybe 5m?

Reply:
OK, I think that makes sense then, thanks.